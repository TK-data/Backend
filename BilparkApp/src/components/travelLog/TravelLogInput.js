import React from 'react';
import { connect } from 'react-redux';
import { StyleSheet } from 'react-native';
import { View, Text } from 'native-base';
import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
import { registerUserModalVisible } from '../../actions/registerUser';
import GooglePlacesInputFrom from './GooglePlacesAutocompleteFrom';
import GooglePlacesInputTo from './GooglePlacesAutocompleteTo';


const styles = StyleSheet.create({
  keyboard: {
    backgroundColor: '#002776',
  },
  logo: {
    height: 25,
    width: 112,
  },
});

class TravelLogInput extends React.Component {

  componentDidMount() {
    this.props.visibleModal(false);
  }

  render() {
    const distance = require('../../../node_modules/react-native-google-matrix/index.js');

    distance.get(
      {
        origin: '-7.841879,110.409193',
        destination: '-7.741194,110.342588',
      },
      (err, data) => {
        if (err) return console.log(err);
        console.log(data);
      },
    );
    return (
      <View>
        <GooglePlacesInputFrom />
        <GooglePlacesInputTo />
        <View>
          <Text>
            test
          </Text>
        </View>
      </View>
    );
  }
}

const mapStateToProps = () => {
  return {
  };
};

const mapDispatchToProps = (dispatch) => {
  return {
    visibleModal: bool => dispatch(registerUserModalVisible(bool)),
  };
};

TravelLogInput.navigationOptions = {
};

export default connect(mapStateToProps, mapDispatchToProps)(TravelLogInput);
