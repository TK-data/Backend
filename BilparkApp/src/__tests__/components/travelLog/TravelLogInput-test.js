import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';

import TravelLogInput from '../../../components/travelLog/TravelLogInput';

const initialState = {
  travelLog: {
    addressFrom: '',
    addressTo: '',
    cargoValue: {
      Cargo: false,
      Comment: '',
    },
    datepickerDate: '15.5.2018',
    datepickerVisible: false,
    distance: '0km',
    formType: true,
    formValue: {
      Passenger: '0',
      Passenger1: '',
      Passenger2: '',
      Passenger3: '',
      Passenger4: '',
      Passenger5: '',
    },
    positionFrom: '0',
    positionTo: '0',
  },
};

const middlewares = [thunk];
const mockStore = configureStore(middlewares);

describe('Testing TravelLogInput', () => {
  it('Renders as expected', () => {
    const comp = shallow(
      <TravelLogInput />,
      { context: { store: mockStore(initialState) } },
    );
    expect(comp.dive()).toMatchSnapshot();
  });
});
