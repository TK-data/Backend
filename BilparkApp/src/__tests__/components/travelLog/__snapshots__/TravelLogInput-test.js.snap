// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TravelLogInput Renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TravelLogInput
    datepickerDate="15.5.2018"
    datepickerVisibility={[Function]}
    datepickerVisible={false}
    distance="0km"
    from="0"
    isLoading={undefined}
    postTravelLog={[Function]}
    resetComponent={[Function]}
    saveDatepickerDate={[Function]}
    saveDistance={[Function]}
    success={undefined}
    to="0"
    travelLog={
      Object {
        "addressFrom": "",
        "addressTo": "",
        "cargoValue": Object {
          "Cargo": false,
          "Comment": "",
        },
        "datepickerDate": "15.5.2018",
        "datepickerVisible": false,
        "distance": "0km",
        "formType": true,
        "formValue": Object {
          "Passenger": "0",
          "Passenger1": "",
          "Passenger2": "",
          "Passenger3": "",
          "Passenger4": "",
          "Passenger5": "",
        },
        "positionFrom": "0",
        "positionTo": "0",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Connect(GooglePlacesInputFrom) />,
        <Connect(GooglePlacesInputTo) />,
        <Styled(ViewNB)
          virtual={undefined}
        >
          <Styled(Text)
            style={
              Object {
                "color": "#fff",
                "fontSize": 18,
                "marginLeft": 10,
                "marginTop": 10,
              }
            }
            virtual={undefined}
          >
            Distanse: 
            0km
          </Styled(Text)>
          <Styled(ViewNB)
            style={
              Object {
                "alignSelf": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "width": 730,
              }
            }
            virtual={undefined}
          >
            <Styled(Text)
              style={
                Object {
                  "alignSelf": "center",
                  "color": "#fff",
                  "fontSize": 18,
                  "fontWeight": "normal",
                }
              }
              virtual={undefined}
            >
              Dato:
            </Styled(Text)>
            <CustomDatePickerIOS
              cancelTextIOS="Cancel"
              confirmTextIOS="Confirm"
              date={2018-05-15T13:52:13.216Z}
              isVisible={false}
              mode="date"
              neverDisableConfirmIOS={false}
              onCancel={[Function]}
              onConfirm={[Function]}
              onDateChange={[Function]}
              onHideAfterConfirm={[Function]}
              reactNativeModalPropsIOS={Object {}}
              titleIOS="Pick a date"
            />
            <Styled(Button)
              light={true}
              onPress={[Function]}
              style={
                Object {
                  "alignSelf": "flex-end",
                }
              }
              virtual={undefined}
            >
              <Styled(Text)
                style={
                  Object {
                    "color": "#000",
                    "fontSize": 20,
                    "fontWeight": "normal",
                    "textDecorationLine": "underline",
                  }
                }
                virtual={undefined}
              >
                15.5.2018
              </Styled(Text)>
            </Styled(Button)>
          </Styled(ViewNB)>
        </Styled(ViewNB)>,
        <Connect(TravelLogPassengerForm) />,
        <Connect(TravelLogCargoForm) />,
        <Styled(Button)
          bordered={true}
          light={true}
          onPress={[Function]}
          style={
            Object {
              "alignSelf": "center",
              "marginBottom": 10,
            }
          }
          virtual={undefined}
        >
          <Styled(Text)
            virtual={undefined}
          >
            Lagre kj√∏ring
          </Styled(Text)>
        </Styled(Button)>,
        null,
      ],
      "style": Object {
        "flex": 1,
        "justifyContent": "center",
      },
      "virtual": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Styled(Text)
              style={
                Object {
                  "color": "#fff",
                  "fontSize": 18,
                  "marginLeft": 10,
                  "marginTop": 10,
                }
              }
              virtual={undefined}
            >
              Distanse: 
              0km
            </Styled(Text)>,
            <Styled(ViewNB)
              style={
                Object {
                  "alignSelf": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "width": 730,
                }
              }
              virtual={undefined}
            >
              <Styled(Text)
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "#fff",
                    "fontSize": 18,
                    "fontWeight": "normal",
                  }
                }
                virtual={undefined}
              >
                Dato:
              </Styled(Text)>
              <CustomDatePickerIOS
                cancelTextIOS="Cancel"
                confirmTextIOS="Confirm"
                date={2018-05-15T13:52:13.216Z}
                isVisible={false}
                mode="date"
                neverDisableConfirmIOS={false}
                onCancel={[Function]}
                onConfirm={[Function]}
                onDateChange={[Function]}
                onHideAfterConfirm={[Function]}
                reactNativeModalPropsIOS={Object {}}
                titleIOS="Pick a date"
              />
              <Styled(Button)
                light={true}
                onPress={[Function]}
                style={
                  Object {
                    "alignSelf": "flex-end",
                  }
                }
                virtual={undefined}
              >
                <Styled(Text)
                  style={
                    Object {
                      "color": "#000",
                      "fontSize": 20,
                      "fontWeight": "normal",
                      "textDecorationLine": "underline",
                    }
                  }
                  virtual={undefined}
                >
                  15.5.2018
                </Styled(Text)>
              </Styled(Button)>
            </Styled(ViewNB)>,
          ],
          "virtual": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                "Distanse: ",
                "0km",
              ],
              "style": Object {
                "color": "#fff",
                "fontSize": 18,
                "marginLeft": 10,
                "marginTop": 10,
              },
              "virtual": undefined,
            },
            "ref": null,
            "rendered": Array [
              "Distanse: ",
              "0km",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <Styled(Text)
                  style={
                    Object {
                      "alignSelf": "center",
                      "color": "#fff",
                      "fontSize": 18,
                      "fontWeight": "normal",
                    }
                  }
                  virtual={undefined}
                >
                  Dato:
                </Styled(Text)>,
                <CustomDatePickerIOS
                  cancelTextIOS="Cancel"
                  confirmTextIOS="Confirm"
                  date={2018-05-15T13:52:13.216Z}
                  isVisible={false}
                  mode="date"
                  neverDisableConfirmIOS={false}
                  onCancel={[Function]}
                  onConfirm={[Function]}
                  onDateChange={[Function]}
                  onHideAfterConfirm={[Function]}
                  reactNativeModalPropsIOS={Object {}}
                  titleIOS="Pick a date"
                />,
                <Styled(Button)
                  light={true}
                  onPress={[Function]}
                  style={
                    Object {
                      "alignSelf": "flex-end",
                    }
                  }
                  virtual={undefined}
                >
                  <Styled(Text)
                    style={
                      Object {
                        "color": "#000",
                        "fontSize": 20,
                        "fontWeight": "normal",
                        "textDecorationLine": "underline",
                      }
                    }
                    virtual={undefined}
                  >
                    15.5.2018
                  </Styled(Text)>
                </Styled(Button)>,
              ],
              "style": Object {
                "alignSelf": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "width": 730,
              },
              "virtual": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Dato:",
                  "style": Object {
                    "alignSelf": "center",
                    "color": "#fff",
                    "fontSize": 18,
                    "fontWeight": "normal",
                  },
                  "virtual": undefined,
                },
                "ref": null,
                "rendered": "Dato:",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "cancelTextIOS": "Cancel",
                  "confirmTextIOS": "Confirm",
                  "date": 2018-05-15T13:52:13.216Z,
                  "isVisible": false,
                  "mode": "date",
                  "neverDisableConfirmIOS": false,
                  "onCancel": [Function],
                  "onConfirm": [Function],
                  "onDateChange": [Function],
                  "onHideAfterConfirm": [Function],
                  "reactNativeModalPropsIOS": Object {},
                  "titleIOS": "Pick a date",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Styled(Text)
                    style={
                      Object {
                        "color": "#000",
                        "fontSize": 20,
                        "fontWeight": "normal",
                        "textDecorationLine": "underline",
                      }
                    }
                    virtual={undefined}
                  >
                    15.5.2018
                  </Styled(Text)>,
                  "light": true,
                  "onPress": [Function],
                  "style": Object {
                    "alignSelf": "flex-end",
                  },
                  "virtual": undefined,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "15.5.2018",
                    "style": Object {
                      "color": "#000",
                      "fontSize": 20,
                      "fontWeight": "normal",
                      "textDecorationLine": "underline",
                    },
                    "virtual": undefined,
                  },
                  "ref": null,
                  "rendered": "15.5.2018",
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bordered": true,
          "children": <Styled(Text)
            virtual={undefined}
          >
            Lagre kj√∏ring
          </Styled(Text)>,
          "light": true,
          "onPress": [Function],
          "style": Object {
            "alignSelf": "center",
            "marginBottom": 10,
          },
          "virtual": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Lagre kj√∏ring",
            "virtual": undefined,
          },
          "ref": null,
          "rendered": "Lagre kj√∏ring",
          "type": [Function],
        },
        "type": [Function],
      },
      null,
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Connect(GooglePlacesInputFrom) />,
          <Connect(GooglePlacesInputTo) />,
          <Styled(ViewNB)
            virtual={undefined}
          >
            <Styled(Text)
              style={
                Object {
                  "color": "#fff",
                  "fontSize": 18,
                  "marginLeft": 10,
                  "marginTop": 10,
                }
              }
              virtual={undefined}
            >
              Distanse: 
              0km
            </Styled(Text)>
            <Styled(ViewNB)
              style={
                Object {
                  "alignSelf": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "width": 730,
                }
              }
              virtual={undefined}
            >
              <Styled(Text)
                style={
                  Object {
                    "alignSelf": "center",
                    "color": "#fff",
                    "fontSize": 18,
                    "fontWeight": "normal",
                  }
                }
                virtual={undefined}
              >
                Dato:
              </Styled(Text)>
              <CustomDatePickerIOS
                cancelTextIOS="Cancel"
                confirmTextIOS="Confirm"
                date={2018-05-15T13:52:13.216Z}
                isVisible={false}
                mode="date"
                neverDisableConfirmIOS={false}
                onCancel={[Function]}
                onConfirm={[Function]}
                onDateChange={[Function]}
                onHideAfterConfirm={[Function]}
                reactNativeModalPropsIOS={Object {}}
                titleIOS="Pick a date"
              />
              <Styled(Button)
                light={true}
                onPress={[Function]}
                style={
                  Object {
                    "alignSelf": "flex-end",
                  }
                }
                virtual={undefined}
              >
                <Styled(Text)
                  style={
                    Object {
                      "color": "#000",
                      "fontSize": 20,
                      "fontWeight": "normal",
                      "textDecorationLine": "underline",
                    }
                  }
                  virtual={undefined}
                >
                  15.5.2018
                </Styled(Text)>
              </Styled(Button)>
            </Styled(ViewNB)>
          </Styled(ViewNB)>,
          <Connect(TravelLogPassengerForm) />,
          <Connect(TravelLogCargoForm) />,
          <Styled(Button)
            bordered={true}
            light={true}
            onPress={[Function]}
            style={
              Object {
                "alignSelf": "center",
                "marginBottom": 10,
              }
            }
            virtual={undefined}
          >
            <Styled(Text)
              virtual={undefined}
            >
              Lagre kj√∏ring
            </Styled(Text)>
          </Styled(Button)>,
          null,
        ],
        "style": Object {
          "flex": 1,
          "justifyContent": "center",
        },
        "virtual": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Styled(Text)
                style={
                  Object {
                    "color": "#fff",
                    "fontSize": 18,
                    "marginLeft": 10,
                    "marginTop": 10,
                  }
                }
                virtual={undefined}
              >
                Distanse: 
                0km
              </Styled(Text)>,
              <Styled(ViewNB)
                style={
                  Object {
                    "alignSelf": "center",
                    "flex": 1,
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "width": 730,
                  }
                }
                virtual={undefined}
              >
                <Styled(Text)
                  style={
                    Object {
                      "alignSelf": "center",
                      "color": "#fff",
                      "fontSize": 18,
                      "fontWeight": "normal",
                    }
                  }
                  virtual={undefined}
                >
                  Dato:
                </Styled(Text)>
                <CustomDatePickerIOS
                  cancelTextIOS="Cancel"
                  confirmTextIOS="Confirm"
                  date={2018-05-15T13:52:13.216Z}
                  isVisible={false}
                  mode="date"
                  neverDisableConfirmIOS={false}
                  onCancel={[Function]}
                  onConfirm={[Function]}
                  onDateChange={[Function]}
                  onHideAfterConfirm={[Function]}
                  reactNativeModalPropsIOS={Object {}}
                  titleIOS="Pick a date"
                />
                <Styled(Button)
                  light={true}
                  onPress={[Function]}
                  style={
                    Object {
                      "alignSelf": "flex-end",
                    }
                  }
                  virtual={undefined}
                >
                  <Styled(Text)
                    style={
                      Object {
                        "color": "#000",
                        "fontSize": 20,
                        "fontWeight": "normal",
                        "textDecorationLine": "underline",
                      }
                    }
                    virtual={undefined}
                  >
                    15.5.2018
                  </Styled(Text)>
                </Styled(Button)>
              </Styled(ViewNB)>,
            ],
            "virtual": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  "Distanse: ",
                  "0km",
                ],
                "style": Object {
                  "color": "#fff",
                  "fontSize": 18,
                  "marginLeft": 10,
                  "marginTop": 10,
                },
                "virtual": undefined,
              },
              "ref": null,
              "rendered": Array [
                "Distanse: ",
                "0km",
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <Styled(Text)
                    style={
                      Object {
                        "alignSelf": "center",
                        "color": "#fff",
                        "fontSize": 18,
                        "fontWeight": "normal",
                      }
                    }
                    virtual={undefined}
                  >
                    Dato:
                  </Styled(Text)>,
                  <CustomDatePickerIOS
                    cancelTextIOS="Cancel"
                    confirmTextIOS="Confirm"
                    date={2018-05-15T13:52:13.216Z}
                    isVisible={false}
                    mode="date"
                    neverDisableConfirmIOS={false}
                    onCancel={[Function]}
                    onConfirm={[Function]}
                    onDateChange={[Function]}
                    onHideAfterConfirm={[Function]}
                    reactNativeModalPropsIOS={Object {}}
                    titleIOS="Pick a date"
                  />,
                  <Styled(Button)
                    light={true}
                    onPress={[Function]}
                    style={
                      Object {
                        "alignSelf": "flex-end",
                      }
                    }
                    virtual={undefined}
                  >
                    <Styled(Text)
                      style={
                        Object {
                          "color": "#000",
                          "fontSize": 20,
                          "fontWeight": "normal",
                          "textDecorationLine": "underline",
                        }
                      }
                      virtual={undefined}
                    >
                      15.5.2018
                    </Styled(Text)>
                  </Styled(Button)>,
                ],
                "style": Object {
                  "alignSelf": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "width": 730,
                },
                "virtual": undefined,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Dato:",
                    "style": Object {
                      "alignSelf": "center",
                      "color": "#fff",
                      "fontSize": 18,
                      "fontWeight": "normal",
                    },
                    "virtual": undefined,
                  },
                  "ref": null,
                  "rendered": "Dato:",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "cancelTextIOS": "Cancel",
                    "confirmTextIOS": "Confirm",
                    "date": 2018-05-15T13:52:13.216Z,
                    "isVisible": false,
                    "mode": "date",
                    "neverDisableConfirmIOS": false,
                    "onCancel": [Function],
                    "onConfirm": [Function],
                    "onDateChange": [Function],
                    "onHideAfterConfirm": [Function],
                    "reactNativeModalPropsIOS": Object {},
                    "titleIOS": "Pick a date",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Styled(Text)
                      style={
                        Object {
                          "color": "#000",
                          "fontSize": 20,
                          "fontWeight": "normal",
                          "textDecorationLine": "underline",
                        }
                      }
                      virtual={undefined}
                    >
                      15.5.2018
                    </Styled(Text)>,
                    "light": true,
                    "onPress": [Function],
                    "style": Object {
                      "alignSelf": "flex-end",
                    },
                    "virtual": undefined,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "15.5.2018",
                      "style": Object {
                        "color": "#000",
                        "fontSize": 20,
                        "fontWeight": "normal",
                        "textDecorationLine": "underline",
                      },
                      "virtual": undefined,
                    },
                    "ref": null,
                    "rendered": "15.5.2018",
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bordered": true,
            "children": <Styled(Text)
              virtual={undefined}
            >
              Lagre kj√∏ring
            </Styled(Text)>,
            "light": true,
            "onPress": [Function],
            "style": Object {
              "alignSelf": "center",
              "marginBottom": 10,
            },
            "virtual": undefined,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Lagre kj√∏ring",
              "virtual": undefined,
            },
            "ref": null,
            "rendered": "Lagre kj√∏ring",
            "type": [Function],
          },
          "type": [Function],
        },
        null,
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
    },
  },
}
`;
